source/simulator/core.cc

```diff
@@ -1711,6 +1713,12 @@ namespace aspect
     {
       TimerOutput::Scope timer (computing_timer, "Refine mesh structure, part 2");
 
+      parallel::distributed::SolutionTransfer<dim,LinearAlgebra::BlockVector>
+      system_trans(dof_handler);
```

Comment:
I think this local variable needs to be deleted as it shadows the one above.

Reply:
Yes, exactly. I put the declarations into the wrong scope. Let's see what happens now.